app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="l-constrained elsewhere__container">
+  <a class="elsewhere__item promo__link" href="<%= I18n.t('stamp_duty.elsewhere.scotland.link') %>">
+    <div class="elsewhere__item-content">
+      <h3 class="promo__heading"><%= I18n.t('stamp_duty.elsewhere.scotland.title') %></h3>
+      <p class="promo__content"><%= I18n.t('stamp_duty.elsewhere.scotland.body') %></p>
+    </div>
+    <svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--arrow-right elsewhere__arrow" focusable="false">
+      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--arrow-right"></use>
+    </svg>
+  </a>
+  <a class="elsewhere__item promo__link" href="<%= I18n.t('stamp_duty.elsewhere.wales.link') %>">
+    <div class="elsewhere__item-content">
+      <h3 class="promo__heading"><%= I18n.t('stamp_duty.elsewhere.wales.title') %></h3>
+      <p class="promo__content"><%= I18n.t('stamp_duty.elsewhere.wales.body') %></p>
+    </div>
+    <svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--arrow-right elsewhere__arrow" focusable="false">
+      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--arrow-right"></use>
+    </svg>
+  </a>
+</div>
```

Comment:
There is a lot of duplicated code in this view and in the views of the same name in the land-and-transaction-tax and land-and-buildings-transaction-tax folders. Is it possible to use the same markup and pass variables to it? For example, this file could be in the shared folder for easier access and the relevant controllers could pass the countries to the calling view.

Reply:
@margOnline I've done something along the lines you have suggested: could you have a look at the last commit on this?

Reply:
This is good, but there are still 2 'links_to_other_tools' partials. Is there a time limit on this pr? I can have a look at using just one partial.

Reply:
@margOnline My fault, missed this first time: I deleted land_and_buildings_transaction_taxes/_links_to_other_tools.html.erb and land_transaction_taxes/_links_to_other_tools.html.erb the original one, stamp_duties/_links_to_other_tools.html.erb, was no longer being used but i forgot to delete it. I have now deleted that as well. All are now replaced with shared/_links_to_other_tools.html.erb.